packages/kit-bg/src/services/ServiceBootstrap.ts

```diff
@@ -225,7 +225,7 @@ export default class ServiceBootstrap extends ServiceBase {
       const onekeyRpc = Object.values(getPresetNetworks())
         .find((item) => item.id === n.id)
         ?.rpcURLs?.find((rpc) =>
-          rpc.url?.startsWith('https://node.onekey.so'),
```

Comment:
![WX20230809-202910](https://github.com/OneKeyHQ/app-monorepo/assets/5107241/527f13e8-a229-4af2-99be-0719306c9877)

I can confirm this doesn't affect the result. Electron always gives full URL (with trailing slash if no extra path in the URL).